juntagrico/templates/menu/admin_menu.html

```diff
@@ -87,14 +94,14 @@ <h5>
             </a>
             <div class="collapse" id="extrasubman">
                 <ul class="nav flex-column subadmin-menu">
-                    {% if operation_group and has_extra_subscriptions %}
+                    {% if perms.juntagrico.is_operations_group and b_has_extra_subscriptions %}
```

Comment:
This was here as well, but exactly the same check is done above and there is no need to check again.
There are some more of these.